terps/tads/glk/osportable.cc

```diff
@@ -961,8 +1012,8 @@ os_is_special_file( const char* fname )
     // "..".  (We use OSPATHCHAR instead of '/' though.)
     const char selfWithSep[3] = {'.', OSPATHCHAR, '\0'};
     const char parentWithSep[4] = {'.', '.', OSPATHCHAR, '\0'};
-    if ((strcmp(fname, ".") == 0) or (strcmp(fname, selfWithSep) == 0)) return OS_SPECFILE_SELF;
-    if ((strcmp(fname, "..") == 0) or (strcmp(fname, parentWithSep) == 0)) return OS_SPECFILE_PARENT;
+    if ((strcmp(fname, ".") == 0) || (strcmp(fname, selfWithSep) == 0)) return OS_SPECFILE_SELF;
```

Comment:
@awlck The original code builds fine for me with Clang on Windows; can this be reverted so that it deviates from upstream as little as possible?

With that done, I think we're in a good place to get this merged.

Reply:
I've finally gotten around to changing this back. Visual Studio is not happy about it, but it does indeed build with Clang.